### PR TITLE
VACMS-2693 2694 2695 - service locations interactions

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #2693 #2694 #2695 

## Testing done
Visual / Behat

## Screenshots
image
image

## QA steps

- As an admin, go to `node/add/health_care_local_health_service`
- Add a second service_location paragraph
- Toggle the first and second instances of `Use facility address?` and confirm they show and hide the corresponding address fieldset appropriately.
- Visually verify that the `Phone` fieldset paragraph in service_locations paragraph is collapsed
- Visually verify that `Use main facility phone number?` field in the phone fieldset in service_location paragraphs is not required.

### CODE REVIEW

**Comment by kevwalsh** [comment]
@ethanteague PR fail for #2695.  

Change the default paragraph type to "none". 

But while you're in there, can you take care of the rest of this config? 

Service_location___VA_gov_CMS

**Comment by kevwalsh** [comment]
@ethanteague even though the switch to a collapsible "Details" fieldgroup was not the intention of #2695 , i think it's a good improvement. while you're on admin/structure/paragraphs_type/service_location/form-display can you also apply it to Address and Service Hours?

**Comment by kevwalsh** [comment]
#2693 QA Pass

#2694 QA Pass

**Comment by ethanteague** [comment]
> @ethanteague PR fail for #2695.
> 
> Change the default paragraph type to "none".
> 
> But while you're in there, can you take care of the rest of this config?
> 
> Service_location___VA_gov_CMS

Updated, @kevwalsh :+1:

**Comment by kevwalsh** [comment]
QA Pass on everything.

File: docroot/modules/custom/va_gov_backend/va_gov_backend.module [hunk 1]
```diff
@@ -559,6 +548,10 @@ function va_gov_backend_page_attachments(array &$attachments) {
 
   $attachments['#attached']['library'][] = 'va_gov_backend/alert_form';
   $attachments['#attached']['library'][] = 'va_gov_backend/system_styles';
+  // Nested address in service location paragraphs result in multiple deltas.
+  // Tracking this with states api gets messy.
+  // Bypass and use behaviors directly.
+  $attachments['#attached']['library'][] = 'va_gov_backend/service_location_address';
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_backend/va_gov_backend.module:554]: Nice!

Comment (ethanteague) [reply docroot/modules/custom/va_gov_backend/va_gov_backend.module:554]: Thank you, sir!

**Pull request merged**